### DISCUSSION TITLE
Using the API Client Side

### DISCUSSION THREAD

**Discussion by ghost** [weather-gov/api#421] [Q&A] [answered]
Is use of this API on the client side supported? For example, can we load the API via JavaScript on the client side to retrieve active alerts on a page of our site, or will there be a rate limit issue? I recognize there are many reasons most implementations of the API are done server side, but curious if we can use it on a public website client side. The primary motivation for querying client side is to ensure data is as fresh as possible.

**Comment by scadergit** [comment, marked as answer]
Yes, that is welcomed; there is no condition on how the data is used. There are rate limits, but that is governed separately by the IP of each end user of your client side application.
